doc/reference/reference_lua/datetime.rst

```diff
@@ -1012,3 +1036,148 @@ Support for relational operators for ``interval`` objects has been added since :
+necessary to add a number of seconds to UTC. It is theorically possible to have
+a negative leap second (a second removed from UTC), but so far,
+all leap seconds have been positive (a second has been added to UTC). Based
+on what we know about the earth's rotation, it is unlikely that we will ever
+have a negative leap second.
+
+The standard way to count UTC time across a leap second is:
+
+..  code-block:: tarantoolsession
+
+    2011-12-31 23:59:57
+    2011-12-31 23:59:58
+    2011-12-31 23:59:59
+    2011-12-31 23:59:60 <-- leap second
+    2012-01-01 00:00:00
+    2012-01-01 00:00:01
+    2012-01-01 00:00:02
+
+The decision to insert a leap second into UTC is currently the responsibility
+of the International Earth Rotation and Reference Systems Service. The IERS is
+measuring the true earth rotation and determines when a leap second has to be
+inserted. Insertion of a leap second is always scheduled for the end of the
+last day of a month, preferably at the end of June or December, at UTC midnight.
+Announcements whether a leap second is scheduled or not are published by the
+IERS in their `Bulletin C <https://hpiers.obspm.fr/iers/bul/bulc/bulletinc.dat>`__.
+
+`IANA <https://www.iana.org/time-zones>`__ (Internet Assigned Numbers Authority)
+maintains a so-called Olson TZ database and provides a set of time zone
+description files, including daylight saving information for the current year
+as well as for past years. Beside the time zone description files it also
+contains a leapseconds file. The TZDB version of the file is used to
+implement the time zones support properly.
+
+This section describes how ``datetime`` module supports leap seconds:
+
+*   Tarantool includes TZDB (also called ``tzdata``) and uses this database
+    for a correct time zones support. Use Lua module :doc:`/reference/reference_lua/tarantool`
+    to get a used version of ``tzdata``:
+
+    Since: :doc:`3.2.0 </release/3.2.0>`
+
+..  code-block:: tarantoolsession
+
+    tarantool> require('tarantool').build.tzdata_version
+    ---
+    - 2024a
+    ...
+
+*   The function :ref:`datetime.parse() <datetime-parse>` correctly parses
+    an input string with 60 seconds:
+
+..  code-block:: tarantoolsession
+
+    tarantool> datetime.parse('23:12:60', {format ='%H:%M:%S'})
+    ---
+    - 1970-01-01T23:13:00Z
+    - 8
+    ...
+
+*   The functions :ref:`datetime.new() <datetime-new>` and
+    :ref:`datetime.now() <datetime-now>`, :ref:`datetime:set() <datetime-set>`
+    accept a table with seconds equal to 60 seconds:
+
+..  code-block:: tarantoolsession
+
+    tarantool> datetime.new({ sec = 60 })
+    ---
+    - 1970-01-01T00:01:00Z
+    ...
+
+    tarantool> dt = datetime.now()
+    ---
+    ...
+
+    tarantool> dt
+    ---
+    - 2024-08-08T18:31:19.431773+0300
+    ...
+
+    tarantool> dt:set({ sec = 60 })
+    ---
+    - 2024-08-08T18:32:00.431773+0300
+    ...
+
+Limitations
+-----------
+
+The supported date range is from -5879610-06-22 to +5879611-07-11.
```

Comment:
Do we really need the examples below? Adding examples for such corner cases is unnecessary in docs.